coderd/database/migrations/000221_notifications.up.sql

```diff
@@ -50,16 +50,35 @@ CREATE TABLE notification_messages
 CREATE INDEX idx_notification_messages_status ON notification_messages (status);
 
 -- TODO: autogenerate constants which reference the UUIDs
-INSERT INTO notification_templates (id, name, title_template, body_template, "group", actions)
-VALUES ('f517da0b-cdc9-410f-ab89-a86107c420ed', 'Workspace Deleted', E'Workspace "{{.Labels.name}}" deleted',
-        E'Hi {{.UserName}}\n\nYour workspace **{{.Labels.name}}** was deleted.\nThe specified reason was "**{{.Labels.reason}}{{ if .Labels.initiator }} ({{ .Labels.initiator }}){{end}}**".',
-        'Workspace Events', '[
-        {
-            "label": "View workspaces",
-            "url": "{{ base_url }}/workspaces"
-        },
-        {
-            "label": "View templates",
-            "url": "{{ base_url }}/templates"
-        }
-    ]'::jsonb);
+INSERT INTO notification_templates
+    (id, name, title_template, body_template, "group", actions)
+VALUES
+	-- Workspace Deleted
+    ('f517da0b-cdc9-410f-ab89-a86107c420ed',
+     'Workspace Deleted',
+     E'Workspace "{{.Labels.name}}" deleted',
+     E'Hi {{.UserName}}\n\nYour workspace **{{.Labels.name}}** was deleted.\nThe specified reason was "**{{.Labels.reason}}{{ if .Labels.initiator }} ({{ .Labels.initiator }}){{end}}**".',
+     'Workspace Events',
+     '[
+          {
+              "label": "View workspaces",
+              "url": "{{ base_url }}/workspaces"
+          },
+          {
+              "label": "View templates",
+              "url": "{{ base_url }}/templates"
+          }
+      ]'::jsonb),
+	-- Workspace Marked as Dormant
```

Comment:
You should move this out into its own migration; customers who have already run this migration will not get the new template.